testing/test-helper.js

```diff
@@ -152,12 +152,12 @@ const REQUEST_URL = '//localhost:9876/amp4test/request-bank/';
 const userAgent = encodeURIComponent(window.navigator.userAgent);
 
 export function depositRequestUrl(path) {
-  return `${REQUEST_URL}deposit/${userAgent}/${path}`;
+  return `${REQUEST_URL}deposit/${userAgent}-${path}/`;
```

Comment:
BTW: I don't know if others think the same. The function name is a bit confusing to me, not related to this PR. I would image `depositRequestUrl` to deposit the request, while `withdrawRequest` does the opposite. But instead `depositRequestUrl` only returns a proxy request url, not really deposit the request.

Reply:
it's subtle, but that's why it's named `depositRequestUrl` instead of `depositRequest`. 

renamed to `getDepositUrl`